lms/static/sass/mobile/main.scss

```diff
@@ -0,0 +1,4 @@
+body {
+  background: #fff;
+  padding: 20px;
```

Comment:
Make sure to test a bunch of pages with this added - typically I don't love adding global styles to the body object, but in this case it might be necessary.

Reply:
Yeah I first I thought to define a class for body but then i realized we can't use bodyclass block in fragments.

Reply:
Please use theme-color("inverse") instead of white and $baseline instead of 20px. This ensures theming maintains even on mobile.

Reply:
@HarryRein I'm getting `$baseline` undefined error on using `$baseline` here, I tried importing base build but still getting the error, this is what I tried to fix that:
```
@import 'base/build';

// mobile - base
@import 'mobile/main';
```

Do you know how I can fix that?

Reply:
@HarryRein I have fixed it, thanks!